.github/workflows/release.yml

```diff
@@ -131,22 +131,11 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
 
-      - name: Read NPM vault secrets
-        uses: hashicorp/vault-action@v3.0.0
-        with:
-          url: ${{ secrets.VAULT_ADDR }}
-          method: approle
-          roleId: ${{ secrets.VAULT_ROLE_ID }}
-          secretId: ${{ secrets.VAULT_SECRET_ID }}
-          secrets: |
-            secret/jenkins-ci/npmjs/elasticmachine token | NPMJS_TOKEN ;
-            totp/code/npmjs-elasticmachine code | TOTP_CODE
-
       - name: npm publish (only for tag release)
         if: startsWith(github.ref, 'refs/tags')
         run: |-
-          echo "//registry.npmjs.org/:_authToken=${{ env.NPMJS_TOKEN }}" > .npmrc
-          npm publish --otp=${{ env.TOTP_CODE }} --provenance
```

Comment:
Currently the "Publishing access" for the elastic-apm-node npm package is set to:

"Require two-factor authentication and disallow tokens"

![Screenshot 2024-07-12 at 9 12 33 AM](https://github.com/user-attachments/assets/1db9070c-a1aa-43c6-a91f-d1b005470544)

Will that need to change now that `--otp=...` is no longer being used?


This npm publishing doc https://docs.npmjs.com/requiring-2fa-for-package-publishing-and-settings-modification says:

> Require two-factor authentication and disallow tokens
> With this option, a maintainer must have two-factor authentication enabled for their account, **and they must
> publish interactively.** Maintainers will be required to enter 2FA credentials when they perform the publish.
> Automation tokens and granular access tokens cannot be used to publish packages.

So I'm not sure how this worked at all from CI. Perhaps those docs are wrong that "they must publish interactively"?

Reply:
Hey @trentm, let's discuss this offline, I'll poke you in our private GitHub issue.